data/presentations.yaml

```diff
@@ -1,3 +1,146 @@
+  weight: 100
+- title: Operationalizing SPIRE at Square
+  link: https://youtu.be/KP9rME3zw6g
+  youtube: KP9rME3zw6g
+  presenters: Matthew McPherrin
+  organizations: Square
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: How Square deployed SPIRE and Envoy on a bare metal/multicloud hybrid environment
+  weight: 100
+- title: Observability in SPIRE at Scale
+  link: https://youtu.be/nIRy_aI0E5k
+  youtube: nIRy_aI0E5k
+  presenters: Andrew Moore 
+  organizations: Uber
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: How to implement telemetry with SPIRE Agents deployed on over 10,000 nodes
+  weight: 100
+- title: "Lessons Learned: Designing Scalable, PKI-based Authentication With SPIRE"
+  link: https://youtu.be/TOjb_imYuLE
+  youtube: TOjb_imYuLE
+  presenters: Eli Nesterov
+  organizations: ByteDance
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: Why ByteDance decided to use SPIRE and how they are deploying SPIRE
+  weight: 100
+- title: Establishing Trust Across Regulatory Boundaries in Complex, Heterogeneous Infrastructures Using SPIRE
+  link: https://youtu.be/MUFQSD6EmZ8
+  youtube: MUFQSD6EmZ8
+  presenters: Jonathan Oddy
+  organizations: TransferWise
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: How the TransferWise financial services company is using SPIRE to move away from shared secrets and easily establish strong trust between software systems running across different domains
+  weight: 100
+- title: "OPA & SPIRE: Decoupled Authentication & Authorization for the Cloud Native Enterprise"
+  link: https://youtu.be/atbjE2P0dtk
+  youtube: atbjE2P0dtk
+  presenters: Ash Narkar
+  organizations: Styra
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: How you can enable decoupled authentication and authorization with SPIRE and OPA using the go-spiffe v2 library
+  weight: 100
+- title: "Project Updates: SPIRE"
+  link: https://youtu.be/-2FK8YkLh_U
+  youtube: -2FK8YkLh_U
+  presenters: Tyler Julian and Andrew Harding
+  organizations: Uber, Scytale/HPE
+  venue: SPIFFE Community Day
+  category: overview
+  date: 24-Apr-2020
+  language: English
+  description: An update on SPIRE, including 0.10 release highlights and version 2 of the go-spiffe library
+  weight: 100
+- title: "Project Updates: SIG-Spec"
+  link: https://youtu.be/5NJXp6iiMec
+  youtube: 5NJXp6iiMec
+  presenters: Justin Burke
+  organizations: Google
+  venue: SPIFFE Community Day
+  category: advanced-tutorial
+  date: 24-Apr-2020
+  language: English
+  description: An update on the in-progress specification for SPIFFE Federation
+  weight: 100
+- title: The SPIFFE Continuum - The set of real values (SPIFFE update)
```

Comment:
I'd drop this video - it will become stale quickly and isn't really of general interest.